docs/security/proxyuser.md

```diff
@@ -0,0 +1,69 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+<div align=center>
+
+![](../imgs/kyuubi_logo.png)
+
+</div>
+
+# Hadoop Credentials Manager
+
+Kyuubi engine is submitted with SparkSubmit argument `--proxy-user`. Its delegation tokens of hadoop
```

Comment:
What if the engine is launched via keytab?

Reply:
Added description for engine submitted with `spark.kerberos.principal` and `spark.kerberos.keytab`.